app/javascript/controllers/markdown_controller.js

```diff
@@ -0,0 +1,26 @@
+import 'bootstrap-markdown/js/bootstrap-markdown'
+
+import { Controller } from 'stimulus'
+import marked from 'marked'
+
+export default class extends Controller {
+  /*
```

Comment:
This reminds me of other interesting experiment to try: how hard it would be to convert that markdown editor to use ActionText, which probably works better with stimulus.